Lib/fontTools/mtiLib/__init__.py

```diff
@@ -1146,12 +1146,33 @@ def expectendswith(self, s):
 		return line
 
 def build(f, font, tableTag=None):
+	"""Parse a Monotype font layout file.
+
+	A font object must be passed, but this may be a "dummy" font; it is only
+	used for sorting glyph sets when making coverage tables.
```

Comment:
The TTFont object is also used to store the OT layout tables built from the FontDame source